### ISSUE TITLE
Quotes in archive not lining up with emails

### ISSUE THREAD

**Issue by Moegf** [alexander-ding/qotw#3]

**Comment by alexander-ding** [comment]
Yeah. The quotes from the old emails aren't extracted as individual email objects. If you want, you can try and write a script to extract the quotes, but do note that they are not consistently formatted. If not, just close the issue.
